File: comments-blocker-admin/src/main/java/com/whoisacat/freelance/ura/commentsBlockerAdmin/CommentsBlockerAdmin.java
package com.whoisacat.freelance.ura.commentsBlockerAdmin;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableScheduling
@SpringBootApplication
public class CommentsBlockerAdmin {
    public static void main(String[] args) {
        SpringApplication.run(CommentsBlockerAdmin.class, args);
    }
}

File: comments-blocker-admin/src/main/java/com/whoisacat/freelance/ura/commentsBlockerAdmin/CommentsBlockerAdmin.kt
package com.whoisacat.freelance.ura.commentsBlockerAdmin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@EnableScheduling
@SpringBootApplication
class CommentsBlockerAdmin

fun main(args: Array<String>) {
	runApplication<CommentsBlockerAdmin>(*args)
}
